sdk/relay/Azure.ResourceManager.Relay/src/Generated/HybridConnectionData.cs

```diff
@@ -27,15 +27,15 @@ public HybridConnectionData()
         /// <param name="createdOn"> The time the hybrid connection was created. </param>
         /// <param name="updatedOn"> The time the namespace was updated. </param>
         /// <param name="listenerCount"> The number of listeners for this hybrid connection. Note that min : 1 and max:25 are supported. </param>
-        /// <param name="requiresClientAuthorization"> Returns true if client authorization is needed for this hybrid connection; otherwise, false. </param>
+        /// <param name="isClientAuthorizationRequired"> Returns true if client authorization is needed for this hybrid connection; otherwise, false. </param>
         /// <param name="userMetadata"> The usermetadata is a placeholder to store user-defined string data for the hybrid connection endpoint. For example, it can be used to store descriptive data, such as a list of teams and their contact information. Also, user-defined configuration settings can be stored. </param>
         /// <param name="location"> The geo-location where the resource lives. </param>
-        internal HybridConnectionData(ResourceIdentifier id, string name, ResourceType resourceType, SystemData systemData, DateTimeOffset? createdOn, DateTimeOffset? updatedOn, int? listenerCount, bool? requiresClientAuthorization, string userMetadata, AzureLocation? location) : base(id, name, resourceType, systemData)
+        internal HybridConnectionData(ResourceIdentifier id, string name, ResourceType resourceType, SystemData systemData, DateTimeOffset? createdOn, DateTimeOffset? updatedOn, int? listenerCount, bool? isClientAuthorizationRequired, string userMetadata, AzureLocation? location) : base(id, name, resourceType, systemData)
```

Comment:
Prepend  `Relay` as `HybridConnection` is also a generic name

Reply:
fixed